mu-plugins/blocks/horizontal-slider/src/block.js

```diff
@@ -115,19 +117,25 @@ function Block( { items, title } ) {
 					</span>
 				) }
 			</div>
-			<div className="horizontal-slider-wrapper" ref={ outerRef }>
+			<ul className="horizontal-slider-wrapper" ref={ outerRef } role="tablist">
 				{ items.map( ( item ) => (
-					<ScreenShot
-						key={ item.title }
-						{ ...item }
-						width={ `${ width }px` }
-						height={ `${ height }px` }
-						aspectRatio={ aspectRatio }
-						queryString={ `?vpw=${ initialWidth * 10 }&vph=${ initialHeight * 10 }` }
-						isReady={ true }
-					/>
+					<li key={ item.title } role="presentation">
+						<ScreenShot
+							{ ...item }
+							width={ `${ width }px` }
+							height={ `${ height }px` }
+							aspectRatio={ aspectRatio }
+							queryString={ `?vpw=${ initialWidth * 10 }&vph=${ initialHeight * 10 }` }
+							isReady={ true }
+							tagProps={ {
+								'aria-controls': item.title,
+								'aria-selected': 'true',
```

Comment:
When I navigate through the list with Voiceover all the variations are read as selected, which I assume would be very confusing. Are we planning to add a script later which sets the first as selected and swaps the selected state when the user selects another variation?

Reply:
This should be set to false. Good catch. 

It'll be selected true on the theme directory side for the short-term.